### PR TITLE
Fix(All): Change of website name

### PR DESCRIPTION
**Pull request by ztevenx100**
Changes:

- [ALL] Change of website name.

image
image

### CODE REVIEW

File: src/pages/Login.jsx
Comment (CamiloLeonP) [inline src/pages/Login.jsx:1]: Leave the email domain in lower case

**Review by CamiloLeonP — CHANGES_REQUESTED** [review]

**Review by CamiloLeonP — APPROVED** [review]

**Comment by CamiloLeonP** [comment]
LGTM!

File: index.html [hunk 1]
```diff
@@ -2,10 +2,16 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Parkify</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
     <base href="/" />
+    <title>Parkiu</title>
```
Comment (djbautista) [inline index.html:7]:
```suggestion
    <title>Parkiu | Disponibilidad en tiempo real de parqueaderos en tu ciudad</title>
```

**Review by djbautista — CHANGES_REQUESTED** [review]
Thanks, @ztevenx100. I've left one small comment before approving this.

**Review by djbautista — APPROVED** [review]
LGTM!

**Pull request merged**